index.html

```diff
@@ -4,12 +4,27 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="styles/normalize.css">
+    <link rel="stylesheet" href="styles/normalize.css" />
     <link rel="stylesheet" href="styles/styles.css" />
-    <title>Document</title>
+    <link rel="stylesheet" href="styles/header.css" />
+    <title>Maket</title>
   </head>
   <body>
+    <header class="header">
+      <div class="header__container container">
+        <div class="header__logo">
+          <img src="./images/Callmar.png" alt="picture of squid" />
+        </div>
+        <div class="header__btns">
+          <button type="button" class="header__button button">Вход</button>
+          <button
+            type="button"
+            class="header__button header__button-outline button-outline button"
```

Comment:
```suggestion
            class="header__button header__button-outline button-outline"
```
it is independent BLOCK (button-outline)

Reply:
button-outline ?

Reply:
![q1](https://user-images.githubusercontent.com/62384781/174318407-15d2665c-194d-4e69-af4f-abc71dda7d27.png)

Reply:
Is not fixed